### DISCUSSION TITLE
Update docker image digest without latest tag

### DISCUSSION THREAD

**Discussion by dgomesb** [renovatebot/renovate#28629] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab - 37.321.1-full
### Please tell us more about your question or problem

I have the following configuration to reach my server
```
   hostRules: [
      {
         matchHost: process.env.ARTIFACTORY_URL
         username: process.env.ARTIFACTORY_USERNAME,
         password: process.env.ARTIFACTORY_PASSWORD,
      },
```
and this is my Dockerfile:
```
FROM artifactory.url/path/to/image@digest
```

When the image has the tag `latest` renovate is able to suggest the replacement. Everything works fine.
Is it possible to get the `newest` instead, like sorting by date?
 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Not supported

**Reply by dgomesb** [reply]
Thanks.
